src/base.jl

```diff
@@ -19,6 +19,15 @@ function log_evaluation(logger::Logger, performance_evaluation)
     updaterun(logger.service, run, "FINISHED")
 end
 
+function log_evaluation(logger::Logger, performance_evaluation)
```

Comment:
Perhaps this function should close `result_channel` before returning?

Reply:
The `result_channel` is just temporal. When we perform `take!(result_channel)`, the value is retrieved and returned. The garbage collector will take care off the channel when function ends, so it's safe to maintain it open.
Also, we are not looping the `result_channel` in any way, so we don't have to worry about that.

(I can remove the `wait(result_channel)` at line 26 because `take!(result_channel)` will actually wait until the Channel contains a value)